### PR TITLE
Ondrej Michal - Mesh & Optimization settings

### PR DESCRIPTION
**Pull request by OndraMichal**
US-8268:
This part implements set and get functions for Mesh and Optimization settings incl. unit tests:
get_optimization_settings
set_optimization_settings
get_mesh_settings
set_mesh_settings
get_model_info

### CODE REVIEW

**Comment by OndraMichal** [comment]
Excuse the changes in:
Examples/Cantilever/Demo1.py, Examples/main.py, RFEM/initModel.py, and UnitTests/test_MeshGenerationStatistics.py.
Those are just minor adjustment.

**Review by dogukankaratas — APPROVED** [review]
One more thing to discuss about Optimization Settings;

It works fine even without activating the AddOn for optimization. It can cause some problems about licensing in the future. 

Technically, it works well.

File: RFEM/Calculate/meshSettings.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+        'windsimulation_mesh_config_value_terrain_objects_all': None,
+        'windsimulation_mesh_config_value_surrounding_model': None,
+        'windsimulation_mesh_config_value_surrounding_model_ifc_objects_id': None,
+        'windsimulation_mesh_config_value_surrounding_model_ifc_objects_all': None,
+        'windsimulation_mesh_config_value_surrounding_model_visual_objects_id': None,
+        'windsimulation_mesh_config_value_surrounding_model_visual_objects_all': None,
+        'windsimulation_mesh_config_value_keep_results_if_mesh_deleted': None,
+        'windsimulation_mesh_config_value_consider_surface_thickness': None,
+        'windsimulation_mesh_config_value_run_rwind_silent': None}
+
```
[the remark this answers is not in the document]
Comment (OndraMichal) [reply RFEM/Calculate/meshSettings.py:101]: Well I can not add default values. I have to differ which parameters are set and which are not. This class is just prototype to show user, which parameters can be set. Frankly, I'm not able to describe them more eloquently than the name of the parameter itself :-)

File: RFEM/enums.py [hunk 2]
```diff
@@ -1205,3 +1205,16 @@ class ModelCheckProcessOptionType(Enum):
     Model Check Process Object Groups Option Type
     '''
     CROSS_LINES, CROSS_MEMBERS, DELETE_UNUSED_NODES, UNITE_NODES_AND_DELETE_UNUSED_NODES = range(4)
+
```
[the remark this answers is not in the document]
Comment (OndraMichal) [reply RFEM/enums.py:1208]: Fiexd by adding this enum into enums.py and unit test.

File: RFEM/Calculate/meshSettings.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dogukankaratas) [reply RFEM/Calculate/meshSettings.py:101]: Maybe adding type of parameters in DocStrings can help us to document this. That looks like my duty 👍🏻

**Pull request merged**